docs/Pflichtenheft.tex

```diff
@@ -451,78 +453,76 @@ \subsubsection{Erstes Starten der Simulation auf virtuellen Maschinen}
   \begin{itemize}[noitemsep]
-    \item In der \gls{Fertigungssimulation} kommt es zu einem Alarmzustand.
+    \item In der \gls{Fertigungssimulation} kommt es zu einem Alarmzustand weil ein Tank droht überzulaufen.
     \item Die \gls{Fertigungssimulation} sendet den entsprechenden Alarm an die registrierte \gls{Uberwachungskonsole}.
     \item Die \gls{Uberwachungskonsole} empf\"angt den Alarm.
-    \item Die \gls{Uberwachungskonsole} zeigt in der \gls{GUI} den Alarm an.
+    \item Die \gls{Uberwachungskonsole} zeigt in der \gls{GUI} den Alarm an, indem die Ampel auf Rot springt und der Kreis neben dem Wert ebenfalls rot wird.
+    \item Da Manuel die Zuflussmengen der Tanks in der \gls{Fertigungssimulation} nicht reduziert, kommt es zu einem Überlauf.
+    \item In der \gls{Fertigungssimulation} erscheint der Text "`Überlauf"' und Manuel klickt auf den "`zurücksetzen"' Knopf.
+    \item Die Fertigungssimulation wird in den Anfangszustand zurück gesetzt.
   \end{itemize}
-  \item[Ausgangssituation] \gls{Fertigungssimulation} und \gls{Uberwachungskonsole} laufen normal weiter. In der \gls{GUI} der
-    \gls{Uberwachungskonsole} ist der Alarmeintrag zu sehen.
+  \item[Ausgangssituation] Die \gls{Uberwachungskonsole} läuft normal weiter und zeigt die zurückgesetzten Werte an. In der \gls{GUI} der
+    \gls{Uberwachungskonsole} ist der Alarmeintrag zu sehen. Die \gls{Fertigungssimulation} ist auf den Anfangszustand zurück gesetzt.
   \item [Nichtfunktionale Anforderungen] Die \gls{GUI} der \gls{Uberwachungskonsole} muss einen empfangenen Alarm nach sp\"atestens
     100ms plus der Aktualisierungsfrequenz anzeigen.
 \end{description}
 
-\subsubsection{Alarm \"andern}
+\subsubsection{\glspl{Simulations-Szenario} verwenden}
 \begin{description}
-  \item[Name] \"Andern eines bereits vorhandenen Alarms in der \gls{Uberwachungskonsole}.
+  \item[Name] Starten eines \gls{Simulations-Szenario}s.
   \item[Akteure] Manuel (Bediener der Software)
-  \item[Eingangsbedingung] Die \gls{Uberwachungskonsole} ist in Betrieb. Es ist mindestens ein Alarm bereits eingeschalten.
+  \item[Eingangsbedingung] Die \gls{Fertigungssimulation} und die \gls{Uberwachungskonsole} sind in Betrieb und miteinander verbunden.
   \item[Ereignisfluss]~\\
   \begin{itemize}[noitemsep]
-    \item Manuel \"offnet in der \gls{Uberwachungskonsole} die Einstellungen.
-    \item Manuel navigiert in den Reiter "`Alarme"'.
-    \item Manuel w\"ahlt den gew\"unschten Alarm aus.
-    \item Manuel tr\"agt den gew\"unschten Schwellenwert ein.
-    \item Manuel klickt auf "`Speichern"'.
+    \item In der \gls{Fertigungssimulation} wird ein \gls{Simulations-Szenario} gestartet.
+    \item Die \glspl{Prozessvariable} ändern sich deterministisch und ohne zutun von Manuel. Diese Änderungen sind in der \gls{Uberwachungskonsole} sichtbar.
+    \item Nachdem das \gls{Simulations-Szenario} fertig ist, wird die \gls{Fertigungssimulation} wieder auf den Anfangswert zurückgesetzt, und das
```

Comment:
Das spart uns auch den Teil mit "Wie stellen wir sicher, dass Endlosschleifen nicht in illegalen Zustaenden der Simulation enden?" :)